webapp/src/main/java/org/opentestsystem/rdw/admin/web/GroupController.java

```diff
@@ -27,9 +31,14 @@
         return service.getByQuery(user, query);
     }
 
+    @PutMapping("/api/groups")
```

Comment:
Hmmm... I would consider soft-deleting an implementation detail that the API should be unaware of, my vote would be for DELETE semantics all the way through.